docs/Configuration.md

```diff
@@ -253,13 +278,15 @@ The ZONEMASTER section has several keys :
 
 ### max_zonemaster_execution_time
 
-An integer.
+A positive integer. Max length 9 digits.
```

Comment:
I think we should limit it further to make testing easier. I do not think we need to support anything longer than 3600 s.